src/support/s3-client.js

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
Should we use spacecat-shared-utils for this?
https://github.com/adobe/spacecat-shared/blob/main/packages/spacecat-shared-utils/src/s3.js